packages/semantic-tokens/scripts/legacyTokens.ts

```diff
@@ -0,0 +1,56 @@
+import fs from 'node:fs';
+import path from 'node:path';
+
+// eslint-disable-next-line no-restricted-imports
+import * as tokensPackage from '@fluentui/tokens';
+import { fluentOverrides } from './fluentOverrides';
+import { format } from 'prettier';
+
+const generateLegacyTokens = () => {
+  console.log('Importing required fluent legacy tokens as flat export');
+
+  const semanticTokenFallbacks = Object.keys(fluentOverrides);
+  const fluentTokens = Object.keys(tokensPackage.tokens);
+  const comment = '// THIS FILE IS GENERATED AS PART OF THE BUILD PROCESS. DO NOT MANUALLY MODIFY THIS FILE\n';
+
+  const generatedTokens = semanticTokenFallbacks.reduce((acc, t) => {
+    const fluent2Fallback = fluentOverrides[t].f2Token;
+    if (!fluent2Fallback) {
+      return '';
+    }
+    if (!fluentTokens.includes(fluent2Fallback)) {
+      // This should never occur, but let's flag if a mistake was made in fallback token names
+      throw new Error(`Fluent token ${fluentOverrides[t].f2Token} not found in fluent tokens`);
+    }
+
+    const token = `/**
+     * CSS custom property value for the {@link @fluentui/tokens#${fluent2Fallback} | \`${fluent2Fallback}\`} design token.
+     * @public
+     */
+    export const ${fluent2Fallback} = 'var(--${fluent2Fallback})';\n`;
```

Comment:
If we had any changes to the value of these tokens, this would break I think? 

ex;
``` typescript
// original
spacingVerticalXXXL: 'var(--spacingVerticalXXXL)',

// becomes
spacingVerticalXXXL: 'var(--minimized-value)',
```

We'd then have the following printed which would be wrong:
`export const spacingVerticalXXXL = 'var(--spacingVerticalXXXL)';\n`;`